BUILDING.md

```diff
@@ -89,7 +89,7 @@ Depending on host platform, the selection of toolchains may vary.
 
 #### Windows
 
-* Visual Studio 2017 or the Build Tools thereof
+* Visual Studio 2017 or the "Build Tools" thereof, on an x64 host.
```

Comment:
Why add quotes?

Reply:
Unless this is supposed to imply that Visual Studio 2017 is not required on a non-x64 host, "on an x64 host" doesn't belong here.

Reply:
It's a proper product name.

Reply:
Visual Studio 2017 is a product name, why isn't it in quotes?